src/automation.ts

```diff
@@ -35,7 +35,10 @@ const processString = (inputStr: string): string => {
 
 (async () => {
     let dynamicQuestions: DynamicQuestion[] = [];
-    const browser: Browser = await chromium.launch();
+    const browser: Browser = await chromium.launch({
```

Comment:
This is now better. Can you also check how to connect to already running browser? For the purpose of this application this is perfectly fine, however to integrate this script with AP we would need to attach to already running tab in the Chromium.